toolkit/scripts/precache.mk

```diff
@@ -34,6 +35,7 @@ $(STATUS_FLAGS_DIR)/precache.flag: $(go-precacher) $(chroot_worker) $(rpms_snaps
 		--snapshot "$(rpms_snapshot)" \
 		--output-dir "$(remote_rpms_cache_dir)" \
 		--output-summary-file "$(precache_downloaded_files)" \
+		--repo-summary-file "$(repo_summary_file)" \
```

Comment:
Small request: can we use a different word than "summary-file". We've been using that word for the JSON files containing the contents of the `RepoContents` structs. Best to avoid overloading terms, if possible.